customer-account-extension/locales/en.default.json

```
@@ -1,3 +1,3 @@
 {
-  "welcome": "Welcome to the {{target}} customer account ui extension!"
+  "welcome": "Welcome to the {{target}} in customer account ui extension!"
```

Will this actually be exposed to 3p devs at some point? If so, I think it could use a bit of re-wording. You may even want to ping someone on the UX team for copy.

```suggestion
  "welcome": "Welcome to the {{target}} target of your in customer account UI extension!"
```

---

> Will this actually be exposed to 3p devs at some point?

Yes. It is only a template to show, and I think 3p devs will surely change it. It is mostly copy paste from [checkout](https://github.com/Shopify/extensions-templates/blob/main/checkout-extension/locales/en.default.json).

> You may even want to ping someone on the UX team for copy.

Makes sense. But I will reach out to ux team next time when they consider multiple templates, and then change it together. Right now is not exposed to 3p, so I think it is ok.

---

Done. Thanks